.github/workflows/ci.yml

```diff
@@ -127,9 +127,10 @@ jobs:
           ./dist/fuseml-installer version
           make fuseml-install-with-extensions
 
-      - name: Run mlflow-e2e
+      - name: Run e2e jobs
         run: |
            # extract the target branch in the context of which this job is running
            branch=${{ github.base_ref }}
            [[ -z "$branch" ]] && branch=${GITHUB_REF#refs/heads/}
            RELEASE_BRANCH=$branch make mlflow-e2e
+           RELEASE_BRANCH=$branch make mlflow-seldon-e2e
```

Comment:
Maybe you should move the seldon test to its own gha job by using a matrix strategy. This would make the test run in parallel and independent.

Reply:
ok